ci/pipeline-staging.yml

```diff
@@ -461,6 +461,51 @@ jobs:
             username: ((slack-username))
             icon_url: ((slack-icon-url))
 
+  - name: nightly-site-bucket-key-rotator
+    plan:
+      - get: src
+        resource: pr-((deploy-env))
```

Comment:
staging's pr resource is just named `pr` (since it runs on release PRs and dev PRs)

Reply:
Updated